src/common/Input/Input.jsx

```diff
@@ -0,0 +1,4 @@
+export default function Input({ labelText, placeholderText, onChange }) {
+	<label>{labelText}</label>;
+	return <input placeholder={placeholderText} onChange={onChange} />;
```

Comment:
I am afraid you `label` will not be populated in final render since it is placed before `return` statement. Fix it, please